Grammar/Grammar

```diff
@@ -17,14 +20,14 @@ decorators: decorator+
 decorated: decorators (classdef | funcdef | async_funcdef)
 
 async_funcdef: 'async' funcdef
-funcdef: 'def' NAME parameters ['->' test] ':' suite
+funcdef: 'def' NAME parameters ['->' test] ':' [TYPE_COMMENT] func_body_suite
```

Comment:
Currently this doesn't give an error, and the type comment recorded is the one on the same line:
```python
def f():  # type: () -> int
    # type: () -> str
    pass
```
I think this should cause a `SyntaxError` (not here, but in `ast.c`).

Reply:
OK, will do.